pkg/rpc/server/testdata/test_contract.go

```diff
@@ -59,18 +63,18 @@ func Transfer(from, to []byte, amount int, data interface{}) bool {
 	return true
 }
 
-func BalanceOf(addr []byte) int {
+func BalanceOf(account interop.Hash160) int {
 	ctx := storage.GetContext()
-	if len(addr) != 20 {
+	if len(account) != 20 {
 		runtime.Log("invalid address")
 		return 0
 	}
 	var amount int
-	val := storage.Get(ctx, addr)
+	val := storage.Get(ctx, account)
 	if val != nil {
 		amount = val.(int)
 	}
-	runtime.Notify("balanceOf", addr, amount)
+	runtime.Notify("balanceOf", account, amount)
```

Comment:
`balanceOf` is a safe method, it's not allowed to notify since 304b9ad3d4cdb76dd043ffdbd99ff30cb9d5cc2d.